CHANGELOG.md

```diff
@@ -22,6 +22,12 @@ IMPROVEMENTS:
 * `azurerm_virtual_machine_scale_set` - Support for updating the customData field [GH-559]
 * `azurerm_virtual_machine_scale_set` - ensuring changes to the `settings` object are detected [GH-609]
 
+## 0.3.4 (December 12, 2017)
+
+IMPROVEMENTS:
+
+* Too Many Request: Upgrade go-autorest to v9.5.2 to keep retrying all the time while meeting 429 status code. ([#617](https://github.com/terraform-providers/terraform-provider-azurerm/pull/617))
+
```

Comment:
@metacpp heads up that we don't manage this file through PR's (also there is no v0.3.4 of the Provider) - so I'm going to push a commit to revert this file/change (and update it as we normally would)